docs/index.rst

```diff
@@ -1,7 +1,12 @@
 django-pgtransaction
 =======================================================================
 
-Welcome to the docs for django-pgtransaction! It doesn't appear that
-the author has created any Sphinx docs for their project yet. Try
-viewing the `README <https://github.com/Opus10/django-pgtransaction>`_
-of their project for documentation.
+django-pgtransaction offers a drop-in replacement for the
+default ``django.db.transaction`` module which, when used on top of a PostgreSQL
+database, extends the functionality of that module with Postgres-specific features.
+
+At present, ``django-pgtransaction`` offers an extension of the
+``django.db.transaction.atomic`` context manager/decorator which allows one to
+dynamically set the isolation level when opening a transaction, as well as specifying
+a retry policy for when an operation in that transaction results in a Postgres locking
+exception. See :ref:`package` and :ref:`example` for more.
```

Comment:
Is the PR missing a file? I'm unable to build the docs.

Since this is a very short library, I'd recommend one of two things:

1. Just put the example directly in the intro or in a "Quickstart" subsection of the intro
2. or remove the package section and just use ``..autofunction:: `` here since the auto docs have nice examples